superset-frontend/src/components/ListView/types.ts

```diff
@@ -89,3 +99,24 @@ export interface FetchDataConfig {
 export interface InternalFilter extends FilterValue {
   Header?: string;
 }
+
+export enum FilterOperators {
+  starts_with = 'sw',
```

Comment:
could we make this snakeCase?

Reply:
converted to camelCase since that's the js standard